benchmarks/src/commonMain/kotlin/BufferOps.kt

```diff
@@ -415,3 +416,67 @@ open class IndexOfByteString {
     @Benchmark
     fun benchmark() = buffer.indexOf(byteString)
 }
+
+@State(Scope.Benchmark)
+open class Utf8CodePointsBenchmark : BufferRWBenchmarkBase() {
+    private val codePointsCount = 128
+
+    // Encoding names follow naming from Utf8StringBenchmark
+    @Param("ascii", "utf8", "sparse", "2bytes", "3bytes", "4bytes", "bad")
+    var encoding: String = "ascii"
```

Comment:
Maybe it's better to use enum in such cases?

Reply:
I decided to use the same values as for the old read/writeString benchmark. I can replace the parameter with enum, but then the old value need to be replaced (especially `2|3|4bytes`), and that would complicate results comparison against older releases.